packages/donation-form/test/tests/modals/upsell-modal-content.test.ts

```diff
@@ -1,6 +1,7 @@
 import { html, fixture, expect, oneEvent } from '@open-wc/testing';
 import { UpsellModalContent, UpsellModalCTAMode } from '../../../src/modals/upsell-modal-content';
 import '../../../src/modals/upsell-modal-content';
+import { html as lit2html } from 'lit';
```

Comment:
This has to be imported differently because the `@open-wc/testing` package still uses the `html` command from Lit1 for its testing, but we need to use the lit2 version for testing.

Reply:
good to know